### PR TITLE
Add kronos ansible inventory

### PR DESCRIPTION
**Pull request by cfarquhar**
- Move the current inventory down one level (`inventory/` -> `inventories/aaronslab`)
- Add a new inventory with the vars used for Kronos
- Update documentation to reflect supporting multiple inventories

### CODE REVIEW

**Comment by cfarquhar** [comment]
run jenkins tests

File: inventories/kronos/group_vars/cloud_snitch_sync.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cloud_snitch_neo4j_enabled: True
+cloud_snitch_neo4j_password: "{{ pwsafe_neo4j_password }}"
+cloud_snitch_neo4j_username: "{{ pwsafe_neo4j_username }}"
+cloud_snitch_neo4j_uri: bolt://172.16.10.5
```
Comment (absalon-james) [inline inventories/kronos/group_vars/cloud_snitch_sync.yml]: We probably should keep actual locations out of inventory also. Maybe this should be an address from host vars of the neo4j instance.

File: inventories/kronos/group_vars/web.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cloud_snitch_web_neo4j_username: "{{ pwsafe_neo4j_username }}"
+cloud_snitch_web_neo4j_password: "{{ pwsafe_neo4j_password }}"
+cloud_snitch_web_neo4j_uri: "bolt://172.16.10.5"
```
Comment (absalon-james) [inline inventories/kronos/group_vars/web.yml]: Same thing here.

File: inventories/kronos/host_vars/neo4j-01.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+ansible_host: 172.16.10.5
+neo4j_default_listen_address: 172.16.10.5
```
Comment (absalon-james) [inline inventories/kronos/host_vars/neo4j-01.yml]: Also here

File: inventories/kronos/group_vars/cloud_snitch_sync.yml (hunk 1 above)
Comment (cfarquhar) [reply inventories/kronos/group_vars/cloud_snitch_sync.yml]: Good feedback.  Latest push fixes this.

File: inventories/kronos/group_vars/web.yml (hunk 2 above)
Comment (cfarquhar) [reply inventories/kronos/group_vars/web.yml]: Good feedback.  Latest push fixes this.

File: inventories/kronos/host_vars/neo4j-01.yml (hunk 3 above)
Comment (cfarquhar) [reply inventories/kronos/host_vars/neo4j-01.yml]: Good feedback.  Latest push fixes this.

**Comment by absalon-james** [comment]
LGTM

**Pull request merged**